documentation/src/docs/deployment/run-local.md

```diff
@@ -69,23 +70,69 @@ Download the [docker-compose.yml](https://github.com/pravega/pravega/tree/master
 wget https://raw.githubusercontent.com/pravega/pravega/master/docker/compose/docker-compose.yml
 ```
 
-We need to set the IP address of our local machine as the value of HOST_IP in the following command.
+We need to set the IP address of our local machine as the value of `HOST_IP` in the following command.
 ```
 HOST_IP=1.2.3.4 docker-compose up
 ```
 Clients can then connect to the controller at `${HOST_IP}:9090`.
 
 ## Configuring standalone
 
-The configuration properties for standalone can be specified in `config/standalone-config.properties`.
+Configure standalone server to communicate using SSL/TLS. To do so, edit the TLS-related properties in `standalone-config.properties` as shown below:
 
-These properties include ports for zookeeper, segment store and controller. They also contain other configurations related to security.
+  ```java
+  singlenode.enableTls=true
+  singlenode.keyFile=../config/key.pem
+  singlenode.certFile=../config/cert.pem
+  singlenode.keyStoreJKS=../config/standalone.keystore.jks
+  singlenode.keyStoreJKSPasswordFile=../config/standalone.keystore.jks.passwd
+  singlenode.trustStoreJKS=../config/standalone.truststore.jks
+
+  ```
+These properties include ports for Zookeeper, Segment Store and Controller. They also contain other configurations related to security.
 
 ## Running standalone with encryption and authentication enabled
```

Comment:
Should the title be something like "Running Pravega in standalone mode with SSL/TLS enabled?"
* The term `standalone` is too general a term. What we are referring to here is `Pavega in standalone mode` (borrowing from https://github.com/pravega/pravega/blob/master/documentation/src/docs/getting-started.md). 
* Everything we are talking about in this section is only related to SSL/TLS. 

Maybe we should all add another section at the same level titled: "Running Pravega in standalone mode with Client Authentication and Authorization enabled", adding auth related configurations.

Reply:
@ravisharda Edited